src/main/java/ecs/PointLight.java

```diff
@@ -11,7 +9,7 @@
  *
  * @author VoxelRifts
  */
-public class PointLight extends RenderableComponent<QuadRenderBatch> {

```

Comment:
by removing this, it does no longer have the update order defined in RenderableComponent - make sure to set it in each constructor loosing this class as superclass

Reply:
Oh ok

Reply:
its the "DRAW" enum thingy

Reply:
lemme fix this up then